src/components/TenantTable.tsx

```diff
@@ -61,38 +73,44 @@ const TenantTable: React.FC<TenantTableProps> = ({
             {tenants.map((tenant) => (
               <tr key={tenant.id} className="bg-white">
                 {hasIcon && (
-                  <td className="px-6 py-4">
-                    {tenant.hasMovedIn ? (
+                  <td className="px-8 py-4">
+                    {tenant.status.is_move === "Moved In" ? (
                       <img src="assets/tick.svg" alt="tick" />
                     ) : (
                       <img src="assets/pending.svg" alt="pending" />
                     )}
                   </td>
                 )}
                 <td className="px-6 py-4 whitespace-nowrap font-medium text-gray-900">
-                  {tenant.name}
+                  {`${tenant.user.first_name} ${tenant.user.last_name}`}
                 </td>
                 <td className="px-6 py-4 whitespace-nowrap font-medium text-gray-900">
-                  {tenant.apartment}
+                  {tenant.apartment.name.en}
                 </td>
                 <td className="px-6 py-4 whitespace-nowrap font-medium text-gray-900">
-                  {tenant.tower}
+                  {tenant.apartment.tower.name}
                 </td>
                 <td className="px-6 py-4 whitespace-nowrap font-medium text-gray-900">
-                  {tenant.rent}
+                  {tenant.apartment.rent}
                 </td>
                 <td className="px-6 py-4 whitespace-nowrap font-medium text-gray-900">
-                  {tenant.moveInDate || ""}
+                  {tenant.status.move_in_date || ""}
                 </td>
                 <td className="px-6 py-4 whitespace-nowrap font-medium text-gray-900">
-                  {tenant.lastSignedIn || ""}
+                  {tenant.status.last_sign_date || ""}
                 </td>
                 <td className="px-6 py-4">
                   <div className="flex gap-x-4 items-center justify-between">
-                    <Button label="View Detail" />
+                    <Button
+                      label="View Detail"
+                      onClick={() => handleTenantDetailClick(tenant.id)}
+                      buttonId={tenant.id}
```

Comment:
Is there is a need of passing buttonId? Also please remove this remove prop from `Button` component